RELEASING.md

```diff
@@ -0,0 +1,9 @@
+# Instructions to release a new version
+
+To release a new version of the aws-sigv4-proxy, please follow these steps:
+
+1. Create a release branch for this minor version series, if one does not exist yet. The convention is to name this branch: `release/v<release series>` where release series has the format `<major version>.<minor version>.x`. Example of branch `release/v1.8.x`
+2. From the release branch, update the content of the `VERSION` file in the root of this repository. The convention is to ommit the patch version if that is in 0. Example of content: `1.8` or `1.8.1`. Merge the PR that updates the `VERSION` file. Confirm that the continuous integration workflow will succeed.
```

Comment:
This is just to handle the current state of things. Right now tags and images were created without the patch number.
do you think it is worth changing this pattern moving forward? I don't see any harm in doing that but do you see any risk?

Reply:
Consistency is probably more important than aligning with my particular version of correctness here, so definitely keep doing what has been done before for now.  I'm not sure whether it's worth making a change moving forward, but if you do communication of the change will be important to avoid surprises.

Reply:
Created this issue https://github.com/awslabs/aws-sigv4-proxy/issues/216 in case we want to change that in the future.